packages/ember-runtime/lib/system/string.js

```diff
@@ -49,35 +49,6 @@ const STRING_DECAMELIZE_REGEXP = (/([a-z\d])([A-Z])/g);
 
 const DECAMELIZE_CACHE = new Cache(1000, str => str.replace(STRING_DECAMELIZE_REGEXP, '$1_$2').toLowerCase());
 
-function _fmt(str, formats) {
```

Comment:
`_fmt` is used in `loc`:

https://github.com/emberjs/ember.js/blob/a7518b940908994ae0577973987fd4cb6a5f7bee/packages/ember-runtime/lib/system/string.js#L81-L88

and also referenced in the [documentation for it](https://emberjs.com/api/ember/2.17/classes/String/methods/loc?anchor=loc).

Think we'll need to update these too.